packages/report-server/src/types.ts

```diff
@@ -30,17 +30,13 @@ interface ReportsRequestBody {
   testConfig?: ReportConfig;
   testData?: Record<string, string | number>[];
 }
-interface ReportsRequestHeaders {
-  Authorization: string;
-}
 
 export interface ReportsRequest<
   P = FetchReportParams,
   ResBody = unknown,
   ReqBody = ReportsRequestBody,
-  ReqQuery = FetchReportQuery,
-  Headers = ReportsRequestHeaders,
-> extends Request<P, ResBody, ReqBody, ReqQuery, Headers> {
+  ReqQuery = FetchReportQuery
+> extends Request<P, ResBody, ReqBody, ReqQuery> {
```

Comment:
Ahh so this was just me cleaning up the type definition as `Header` isn't a supported field here. However I should have actually added the `Header` type to the fields of the request, so I'll do that 👍

Reply:
So I double checked this and there's no need to add the `header` definition actually, as we extend from `Request<P, ResBody, ReqBody, ReqQuery>` which already has it

Since there's nothing special about our headers there's not need for a specific header interface just as yet 👍

Reply:
Ahhh, makes sense! Just learning Typescript so forget about how `extends` works. Thanks :)